lib/util.js

```diff
@@ -75,56 +85,26 @@ exports.format = function(f) {
   var str = '';
   var a = 1;
   var lastPos = 0;
-  for (var i = 0; i < f.length;) {
+  var i = 0;
+  while (i < f.length) {
     if (f.charCodeAt(i) === 37/*'%'*/ && i + 1 < f.length) {
-      switch (f.charCodeAt(i + 1)) {
-        case 100: // 'd'
-          if (a >= argLen)
-            break;
-          if (lastPos < i)
-            str += f.slice(lastPos, i);
-          str += Number(arguments[a++]);
-          lastPos = i = i + 2;
-          continue;
-        case 105: // 'i'
-          if (a >= argLen)
-            break;
-          if (lastPos < i)
-            str += f.slice(lastPos, i);
-          str += parseInt(arguments[a++]);
-          lastPos = i = i + 2;
-          continue;
-        case 102: // 'f'
-          if (a >= argLen)
-            break;
-          if (lastPos < i)
-            str += f.slice(lastPos, i);
-          str += parseFloat(arguments[a++]);
-          lastPos = i = i + 2;
-          continue;
-        case 106: // 'j'
-          if (a >= argLen)
-            break;
-          if (lastPos < i)
-            str += f.slice(lastPos, i);
-          str += tryStringify(arguments[a++]);
```

Comment:
It seems this function was used only here (linter reports `'tryStringify' is defined but never used`)